Week1/SantiagoMedina/Gradle/fatJar/vehicleFactory/src/main/java/vehicle/VehicleFactory.java

```diff
@@ -0,0 +1,66 @@
+package vehiclefactory;
+
+import java.io.InputStream;
+import java.util.*; 
+
+public class VehicleFactory {
+
+    public static void main (String[] args){
+
+        Welcome();
```

Comment:
In Java, the first character should be lower case, in .Net, is like you have here.

Reply:
sorry for that method, I thought I had written it in lowercase. i changed it.